From 83fb4f090371f5ec2965a39b00d2c14959f789a8 Mon Sep 17 00:00:00 2001
From: Neta Zur <neta@marvell.com>
Date: Tue, 28 Jul 2015 10:54:56 +0300
Subject: [PATCH 0275/1240] fix: pci: update type of pci_dev_t

For 64 bit set to unsigned long. for 32 bit set to int

Change-Id: Ic5162454fc5a5b109739f41e69a15f48cd2d15da
Signed-off-by: Neta Zur <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/22251
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 include/pci.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/pci.h b/include/pci.h
index 3ef6295..3b3ff73 100644
--- a/include/pci.h
+++ b/include/pci.h
@@ -422,11 +422,11 @@
 #ifdef CONFIG_SYS_PCI_64BIT
 typedef u64 pci_addr_t;
 typedef u64 pci_size_t;
-typedef u64 pci_dev_t;
+typedef unsigned long pci_dev_t;
 #else
 typedef u32 pci_addr_t;
 typedef u32 pci_size_t;
-typedef u32 pci_dev_t;
+typedef int pci_dev_t;
 #endif
 
 struct pci_region {
-- 
1.9.1

